building-a-preview/index.md

```diff
@@ -0,0 +1,777 @@
+infrastructure. This might include things like installing required packages or
+tools, or overriding default configuration files.
+
+During the `update` phase, you might use commands that import data or other
+assets into a service. This might include things like importing a database, or
+syncing image files into a service.
+
+During the `build` phase, you might use commands that build or generate the
+actual site. This might include things like compiling Sass files, updating 3rd
+party libraries, or running database updates that the current code in the
+preview depends on.
+
+When using various [Preview Actions](#preview-actions), the build process may
+bypass phases:
+
+- [Preview Actions that start at `init`](#preview-actions-that-start-at-init)
+- [Preview Actions that start at `update`](#preview-actions-that-start-at-update)
+- [Preview Actions that start at `build`](#preview-actions-that-start-at-build)
+
+> #### Hint:: Troubleshooting Tugboat not pulling changes
+>
+> Have you changed the Docker image in your build, changed the database you're
+> pulling during the `update` phase, or added new resources or commands during
+> `init`? A common cause for changes not appearing in your Tugboat Preview is
+> that the Preview build isn't a brand new build from scratch. For example,
+> building a Preview after [setting a Base Preview](#set-a-base-preview) means
+> that new Previews start from the `build` phase - so if you're changing the
+> database in `update`, or changing the Docker image (which gets pulled before
+> `init`), you won't see your changes in the child Preview. Read through which
+> Preview actions start at which phase to help you troubleshoot whether you need
+> to do a different type of Preview build - i.e.
+> [building a Preview from scratch after setting a Base Preview](#building-a-preview-from-scratch-after-youve-set-a-base-preview).
+
+#### Preview Actions that start at init
+
+Three types of Preview builds start from the very beginning of the process, at
+the `init` phase:
+
+- [Building a new Preview](#how-to-build-a-preview) from scratch (without a Base
+  Preview)
+- [Rebuilding](#rebuild) a Preview without a Base Preview
+- [Rebuilding](#rebuild) a Base Preview
+
+> #### Note::Rebuilding a Preview from a Base Preview
+>
+> If you're
+> [Rebuilding a Preview that was built from a Base Preview](#preview-actions-that-start-at-build),
+> the build starts at the `build` phase - not the `init` phase.
+
+#### Preview Actions that start at update
+
+Two types of Preview builds start from the `update` phase, bypassing `init`:
+
+- [Refreshing](#refresh) a Preview that has no Base Preview
+- [Refreshing](#refresh) a Base Preview
+
+#### Preview Actions that start at build
+
+Two types of Preview builds start from the `build` phase, bypassing `init` and
+`update`:
+
+- Building a new Preview from a Base Preview
+- Rebuilding a Preview that was built from a Base Preview
+
+> #### Hint:: Making changes to child Previews
```

Comment:
We say basically the same thing here as the callout above

Reply:
Yup, I've included callouts in both places in case users follow the jump links to go directly to one of these phases, and don't bother to read the rest. I'm still pondering whether this is the right way to organize and present this info; maybe that diagram I'm working on would alleviate the need to have this text here at all. But then again, I can link to the text from other parts of the docs site. Anywho, if you feel strongly about not repeating this info, I can remove one of the callouts - just let me know.